src/qml/QFieldCloudPopup.qml

```diff
@@ -438,7 +438,17 @@ Popup {
             }
             wrapMode: Text.WordWrap
             horizontalAlignment: Text.AlignHCenter
-            Layout.bottomMargin: 10
+            Layout.fillWidth: true
+          }
+
+          Text {
+            font: Theme.tipFont
+            color: Theme.mainColor
+            text: cloudProjectsModel.currentProjectData.UploadAttachmentsStatus === QFieldCloudProjectsModel.UploadAttachmentsInProgress
+                  ? qsTr( 'Attachments are currently uploaded in the background' )
```

Comment:
Would be nice to have any feedback how much is left, e.g. "20 attachments are currently uploaded". I know they might be 19 1MB pictures and 1 20MB picture, but at least gives some visual feedback what is going on.

Reply:
@suricactus , OK, that's a nice little addition. I don't think there's much use in adding size info for now since it'll likely be images with +/- the same size. A remaining file count is a nice way to keep it simple while informative.

Reply:
@suricactus , done